src/p5.dance.js

```diff
@@ -1116,6 +1120,29 @@ module.exports = class DanceParty {
     }, this);
   }
 
+  // Called when executing the AI block.
+  async ai(value) {
```

Comment:
Could we rename this function? Maybe something like `aiGenerateStage`?

Reply:
This relates to the name of the black and the function it calls in the main repo [here](https://github.com/code-dot-org/code-dot-org/pull/53769/files#diff-539aa11323c94e9339a5d0316f6072470618ce8c67ef57acaf2d792ee87be8caR5).  I agree it should be something more specific, but it doesn't feel like our design is quite crisp enough yet.  As we discussed separately, let's address better naming in the next week as we do the next round of work.